lib/librumpkern_compiler_rt/Makefile

```diff
@@ -0,0 +1,9 @@
+LIB=		rumpkern_compiler_rt
```

Comment:
I wouldn't call it `rumpkern_compiler_rt` since it has nothing to do with the rump kernel.  How just `compiler_rt`?